multiqc/plots/bargraph.py

```diff
@@ -472,8 +460,8 @@ def matplotlib_bargraph(plotdata, plotsamples, pconfig=None):
                         prevdata[i] += prev_values[i]
                 # Default colour index
                 cidx = idx
-                while cidx >= len(default_colors):
-                    cidx -= len(default_colors)
+                while cidx >= len(scale.colours):
+                    cidx -= len(scale.colours)
```

Comment:
I wonder if we can move this logic into `get_colour()` (allowing idx > array length, using remainder).

Reply:
This logic is already there, actually, so can just remove this block.